### DISCUSSION TITLE
A newbee struggling with a workflow errors - Missing Models and Nodes

### DISCUSSION THREAD

**Discussion by jaivishal0902** [Comfy-Org/ComfyUI#5615] [General]
I started recently when stumbled upon amazing videos on YouTube. I clearly see it can give my product photography a new dimension. I am strugling with this attached worlflow since weeks and not able to make it run. I have solved most of it but few nodes/ models are not getting sorted. Attached is the workflow (JV.json), if someone can run it successfully and tell me what all nodes & models i need to install - i would be really thankful.
![Screenshot 2024-11-14 at 8 33 09 PM](https://github.com/user-attachments/assets/b242aec3-c551-4d15-a98e-23bbb54578ec)

[JV.json](https://github.com/user-attachments/files/17750917/JV.json)

**Comment by LukeG89** [comment]
About the nodes, if you have [ComfyUI-Manager](https://github.com/ltdrdata/ComfyUI-Manager) installed, you can easily find and install the missing ones:

![manager](https://github.com/user-attachments/assets/febdd22c-0ccb-462b-adb3-21df1ad07602)


About the models, you can search them on [CivitAI](https://civitai.com/) or [Huggingface](https://huggingface.co/), for example:

[Realistic Vision V5.1](https://civitai.com/models/4201?modelVersionId=130072)

[vae-ft-mse-840000-ema-pruned](https://civitai.com/models/276082/vae-ft-mse-840000-ema-pruned-or-840000-or-840k-sd15-vae)

**Reply by jaivishal0902** [reply]
Have Tried it all with due respect. If you can run it successfully and then comment, would be amazing. Have you been able to run the workflow ? If not then please do.

**Reply by LukeG89** [reply]
I'm missing many custom nodes (and I don't want to install them), so I haven't tried running your workflow.

Why are you asking me to try it? Are you encountering any errors?

**Reply by jaivishal0902** [reply]
I get this error while trying to install via URL

![Screenshot 2024-11-16 at 8 00 19 PM](https://github.com/user-attachments/assets/61e95087-9f4f-481a-a372-d90b44d90e74)

**Reply by jaivishal0902** [reply]
> About the nodes, if you have [ComfyUI-Manager](https://github.com/ltdrdata/ComfyUI-Manager) installed, you can easily find and install the missing ones:
> 
> ![manager](https://private-user-images.githubusercontent.com/161304036/386286813-febdd22c-0ccb-462b-adb3-21df1ad07602.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzE3Njg3ODMsIm5iZiI6MTczMTc2ODQ4MywicGF0aCI6Ii8xNjEzMDQwMzYvMzg2Mjg2ODEzLWZlYmRkMjJjLTBjY2ItNDYyYi1hZGIzLTIxZGYxYWQwNzYwMi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTE2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTExNlQxNDQ4MDNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xNDBkMmUzMmMxMmMzZTZhMTcxMmIxZDY5ZTdjZjA3ODUwNDhjODMxMmM1M2IxMTY4NTM2MzdkNzhjMThjMDRlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.Zi0xzjiv1lhvMSRbAZnx2s3yBnFxDe_MRB8nLzFK6v4)
> 
> About the models, you can search them on [CivitAI](https://civitai.com/) or [Huggingface](https://huggingface.co/), for example:
> 
> [Realistic Vision V5.1](https://civitai.com/models/4201?modelVersionId=130072)
> 
> [vae-ft-mse-840000-ema-pruned](https://civitai.com/models/276082/vae-ft-mse-840000-ema-pruned-or-840000-or-840k-sd15-vae)

Hi Luke - I have manager installed and have tried the Install Missing Nodes many a times and now it doesnt show any missing nodes however when i run queue prompt, it doesnt run and many nodes turn red. Like this Node 330 and 39. I able to fix 30 but 39 doesnt let me select any node (no dropdown). Weird.

![Screenshot 2024-11-16 at 8 20 31 PM](https://github.com/user-attachments/assets/cab1cfa9-1338-4303-ba60-6ef35a2d3bee)
![Screenshot 2024-11-16 at 8 21 04 PM](https://github.com/user-attachments/assets/d67bc31e-6b41-43aa-916b-ac7d0035ad18)

**Reply by jaivishal0902** [reply]
Hi @LukeG89 - As i shared the images above. I do have the manager installed but its of no help. I am at a deadend now. Would you like to try the workflow that i attached whenerv you have sone spare time please ? No hurry and no obligation :-)

**Reply by LukeG89** [reply]
> I able to fix 30 but 39 doesnt let me select any node (no dropdown). Weird.

@jaivishal0902 But have you downloaded the VAE model file (the one I linked in the first comment) and placed in `ComfyUI/models/vae` folder?

**Reply by LukeG89** [reply]
When you install new custom nodes, not all the time the models they use will be downloaded automatically. In this case if you miss some models, you need to check the custom node repositories to see where you can find and place them.

**Comment by keremcozin** [comment]
Hey thank you so much! I really love this community.
